src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
Can you correct indentation in this file?

Reply:
done